### PR TITLE
UI-3301: Load main number's E911 dialog on top notification click at Smart PBX

### PR DESCRIPTION
**Pull request by guillegr123**

### CODE REVIEW

File: submodules/strategy/strategy.js [hunk 1]
```diff
@@ -910,42 +907,33 @@ define(function(require) {
...
-
-								if (_.includes(numberData.features, 'e911')) {
-									e911Active = true;
-									return false;	// Break loop
-								}
+							var isE911Active = _.some(numbers, function(numberData) {
+									return _.includes(numberData.number.features, 'e911');
+								}),
+								e911NumberData
+									= isE911Active
```
Comment (azefiel) [inline submodules/strategy/strategy.js]: Please avoid linebreaks before or after `=` in an assignment.

File: submodules/strategy/strategy.js [hunk 2]
```diff
@@ -910,42 +907,33 @@ define(function(require) {
...
-									callbacks: {
-										success: function(data) {
-											afterFeatureUpdate(e911NumberData.id, data.data.features);
+									monster.ui.toast({
+										type: 'warning',
+										message: self.i18n.active().strategy.toastrMessages.noE911NumberAvailable
+									});
+								} else {
+									monster.pub('common.e911.renderPopup', {
+										phoneNumber: _.get(e911NumberData, 'phoneNumber', e911NumberData.id),
```
Comment (azefiel) [inline submodules/strategy/strategy.js]: I believe that after your refactoring, the number data is accessible at `e911NumberData.number`.

File: submodules/strategy/strategy.js (hunk 1 above)
Comment (guillegr123) [reply submodules/strategy/strategy.js]: :+1: Got it!

File: submodules/strategy/strategy.js (hunk 2 above)
Comment (guillegr123) [reply submodules/strategy/strategy.js]: Indeed, thanks!

**Comment by guillegr123** [comment]
Added success callback handling for action + some refactor to involved functions.

**Pull request merged**